core/src/test/java/tech/tablesaw/api/DoubleColumnTest.java

```diff
@@ -465,4 +465,23 @@ private DoubleColumn createDoubleColumn(double[] originalValues) {
         }
         return initial;
     }
+
+    private boolean computeAndValidateCumSum(double[] originalValues, double[] expectedValues) {
```

Comment:
can you remove the `computeAndValidateCumSum` functions and just put the logic directly in `testCumSum`?  We only create a separate method in tests if it will be called by multiple tests. Otherwise it just adds a bit more code and makes the flow harder to follow